src/videodownlinkbandwidthpolicy/VideoPriorityBasedPolicy.ts

```diff
@@ -552,31 +582,33 @@ export default class VideoPriorityBasedPolicy implements VideoDownlinkBandwidthP
     }
 
     if (this.downlinkStats.packetsLost > 0) {
-      this.setProbeState(RateProbeState.NotProbing);
-      this.logger.info(`bwe: Canceling probe due to network loss`);
-      this.probeFailed = true;
-      this.timeBeforeAllowSubscribeMs =
-        Math.max(
-          VideoPriorityBasedPolicy.MIN_TIME_BETWEEN_SUBSCRIBE_MS,
-          this.timeBeforeAllowSubscribeMs
-        ) * 3;
-      // packet lost indicates bad network and thus slowing down subscribing by extend delay by 3 times
-      return UseReceiveSet.PreProbe;
+      this.logger.info(`bwe: Probe encountering packets lost:${this.downlinkStats.packetsLost}`);
+      // See comment above `VideoPriorityBasedPolicy.SPURIOUS_PACKET_LOST_THRESHOLD`
+      if (
+        this.downlinkStats.packetsLost > VideoPriorityBasedPolicy.SPURIOUS_PACKET_LOST_THRESHOLD
+      ) {
+        this.setProbeState(RateProbeState.NotProbing);
+        this.logger.info(
+          `bwe: Canceling probe due to packets lost:${this.downlinkStats.packetsLost}`
+        );
+        this.probeFailed = true;
+        this.timeBeforeAllowSubscribeMs =
+          Math.max(
+            VideoPriorityBasedPolicy.MIN_TIME_BETWEEN_SUBSCRIBE_MS,
+            this.timeBeforeAllowSubscribeMs
+          ) * 3;
+        // packet lost indicates bad network and thus slowing down subscribing by extend delay by 3 times
+        return UseReceiveSet.PreProbe;
+      }
     }
     const subscribedRate = this.calculateSubscribeRate(this.optimalReceiveStreams);
     if (this.chosenStreamsSameAsLast(chosenStreams) || targetDownlinkBitrate > subscribedRate) {
-      let avgRate = 0;
-      for (const chosenStream of chosenStreams) {
-        avgRate += chosenStream.avgBitrateKbps;
-      }
-      if (targetDownlinkBitrate > avgRate) {
```

Comment:
Is this logic unnecessary? It doesn't seem to be mentioned in the PR description.